script/fetch_content.py

```diff
@@ -4,27 +4,48 @@
 import re
 
 
-def arguments():
+def fetch_arguments():
+  """Fetch arguments from command line"""
+
   parser = argparse.ArgumentParser()
-  parser.add_argument('--url', help='add url')
-  parser.add_argument('--text', help='add text')
-  args = parser.parse_args()
-  return args
-def add_characters():
-  for m in re.finditer(text, r.text):
-    start = m.start()
-    end = m.end()
+  parser.add_argument('url', help='add url')
+  parser.add_argument('search_phrase', help='add text')
+  args = parser.parse_args()  
+  return (args.url, args.search_phrase)
+
+
+def print_10_characters_before_and_after_text(search_phrase, text):
+  """
+  Print text with additional characters.
+
+  This function retrieves the text and returns it with an additional ten characters before and after the text.
+
+  parameters:
+  search_phrase: text that someone enters.
+  r.text: all text on the website.
+  start: index of the first letter of the entered text.
+  end: index of the last letter of the entered text.
+  new_text: a string with additional characters added before start and after end.
+  list_character: list containing elements consisting of 10 characters.
+
+  Return:
+  list character.
+  
+  """ 
+  
+
+  first_occurrence = r.text.find(search_phrase)
+  if first_occurrence >= 0:
+    found_text = re.search(search_phrase, r.text)
```

Comment:
You don't need this :-)
Try to find how to calculate `start` and `end` without using second search (`find` is first).
How to calculate `start` and `end` position using only `first_occurrence` and `search_phrase`?

Try to find out if not I will help you tomorrow :-)

Reply:
I figured it out :)